### ISSUE TITLE
[Feature Request] MPAA, rating + touch the screen to pause/play video

### ISSUE THREAD

**Issue by bronnel** [archos-sa/aos-AVP#1]
Hi, I'm happy now that the project is open source, however I have 2 long awaited feature requests that I want so badly on my favorite video player of all time, 
1-I want to be able Pause / Play Video just by touching the screen (I mean single tap to pause video then single tap to resume) without showing on-screen controls, Yes I know I can use an external player, but I prefer the built-in player of archos, since it is very stable and perfect.
2-Parental guideline rating flags for movies and tv shows
for movies: G -- PG -- PG-13 -- R -- NC-17
for tv shows: TVY -- TVY7 -- TVG -- TVPG -- TV14 -- TVMA

**Comment by bronnel** [comment]
I posted this issue 5 months ago, and no one here even cares, I already built the apk using ubuntu, and android studio,  can you at least tell us how to implent those features?

**Comment by bronnel** [comment]
Fuck you all for not answering

**Comment by bronnel** [comment]
Can someone answer and tell me GFY at least?

**Comment by AKprisonER** [comment]
umm.. I don;t belong to this community. But We can use Exoplayer's On touch listener for Pause and Play to implement those thing.

**Comment by AKprisonER** [comment]
kind of 
 ```
public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_single_contact);

    // getting intent data
    Intent in = getIntent();
    String mobile = in.getStringExtra(TAG_URL);

    Uri uri=Uri.parse(mobile);
    VideoView video=(VideoView)findViewById(R.id.videoView1);
    video.setVideoURI(uri);
    video.start();
    }
```

**Comment by bronnel** [comment]
Yes exactly thank you,  I will try it, but if you have any spare time, can you have a go at it.

**Comment by AKprisonER** [comment]
I have a great difficulty in setting this repo in my PC as I use Windows along with metered connection.
I am stuck in that thing.

**Comment by bronnel** [comment]
I have been using ubuntu for the past several years, I'm  quite familiar with it, I successfully built the repo and apk, but If you don't have access to linux, then you can use android studio on windows, I built the project with it as well, you can clone the github repo from this link
https://github.com/archos-sa/aos-Video
First you need to sign in with your github username and pass, let me know of if you've got any issues.

**Comment by AKprisonER** [comment]
Repo takes about 12.5 gb as stated by one of this project maintainer... does it consumed that much for setting up this repo ?
Btw I hadn't downloaded the Repo tool too. I don't what wil be the total size

**Comment by AKprisonER** [comment]
Also using the link given by you, that can be imported. But as it has no makefile to compile all the dependencies, it is diffult to get it compiled

**Comment by bronnel** [comment]
I don't  remember if it was 12.5 gb but it took a couple of hours to compile the first time,  but I roughly estimate it about that siz 12.5 gb , however can you download repo tool on windows?

**Comment by AKprisonER** [comment]
I don't know about those things. Will give a shot.
Btw have you downloaded tat in ur Ubuntu ?

**Comment by bronnel** [comment]
I don't have that, but I troubleshoot a lot of errors using google until I finally built it

**Comment by AKprisonER** [comment]
Thats nice, But did you imported all other dependencies manually ?
If yes can you share your repo, so that i can built it...
Anyways i wil have less time

**Comment by bronnel** [comment]
No, I used all command line from the main page here. Its ok we can tackle it another time,  I will try my best to give you all the help you need, in a couple of weeks, keep it up.

**Comment by AKprisonER** [comment]
I have to switch from WIN to Ubuntu to get out this mess.

**Comment by bronnel** [comment]
Its really easy if you got any Ubuntu issues I will try to help as much as I can,

**Comment by bronnel** [comment]
I usually install Ubuntu on separate hdd , but you can partition your hdd to have ubuntu installed side by side with your windows, I mean dual boot using grub boot manager

**Comment by AKprisonER** [comment]
Already My Pc is filled with WIN10, MAC os, and Parrot security OS.
Will try this Repo on my Windows. 
WIll let you knw if i get a successful built

**Comment by bronnel** [comment]
Ok, keep it up bro

**Comment by courville** [comment]
Note that repo (git clone of all the repos) and build takes around 1.1GB of data on my disk.

**Comment by bronnel** [comment]
can someone help me to add MPAA rating info to movies and tv shows

**Comment by bronnel** [comment]
@AKprisonER , I just build the project in windows 10 using android studio 3.1, here is how I resolved the dependency issue
1-clone https://github.com/archos-sa/aos-MediaLib  renmae directory to MediaLib (remove "aos-")
2-clone https://github.com/archos-sa/aos-FileCoreLibrary renmae directory to FileCoreLibrary (remove "aos-")
3-clone https://github.com/archos-sa/aos-Video renmae directory to Video (remove "aos-")
thats it

**Comment by AKprisonER** [comment]
do we have place all those file in a single directory ?

**Comment by bronnel** [comment]
yes exactly, and browse to that folder
